_data/communication.yml

```diff
@@ -13,6 +13,13 @@ websites:
       sms: Yes
       doc: https://help.37signals.com/sign-in/questions/460-how-do-i-set-up-and-use-phone-verification-for-logging-in
 
+    - name: Carbon Black Community Exchange
+      url: https://community.carbonblack.com/
```

Comment:
@thegeekkid, Do you think it would be possible to change this link to https://www.carbonblack.com, since the community link redirects to a login page?